package.json

```diff
@@ -103,6 +107,13 @@
     "mssql:test": "DB=mssql npm run plaintest",
     "mssql:destroy": "docker-compose -f scripts/mssql-docker-compose.yml stop",
     "postmssql:init": "node scripts/wait-for-mssql-server.js && npm run mssql:logs || (npm run mssql:logs;false)",
+    "prepublish": "npm run babel",
+    "pre_test": "npm run lint && npm run lint_types",
```

Comment:
duplicate, there's pretest

Reply:
is pretest run automatically before publish?

Reply:
aa you meant pre_test 👍